kubeflow/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018 The Kubeflow Authors All rights reserved.
```

Comment:
What's this docker file for?

Reply:
Just for an e2e example outside kubeflow. I'll remove it once I get fairing baked into our notebook images.